Backport upstream fix to restart irqbalance for cpu-partitioning and realtime profiles.

See: https://github.com/redhat-performance/tuned/pull/243

Do not try to patch these functions, redefine them.
diff -Nura a/usr/lib/tuned/functions b/usr/lib/tuned/functions
--- a/usr/lib/tuned/functions
+++ b/usr/lib/tuned/functions
@@ -621,3 +621,18 @@
 
 	exit $RETVAL
 }
+
+irqbalance_banned_cpus_clear() {
+    sed -i '/^IRQBALANCE_BANNED_CPUS=/d' /etc/sysconfig/irqbalance || return
+    if [ ${1:-restart} = restart ]; then
+        systemctl try-restart irqbalance
+    fi
+}
+
+irqbalance_banned_cpus_setup() {
+    irqbalance_banned_cpus_clear norestart
+    if [ -n "$1" ]; then
+        echo "IRQBALANCE_BANNED_CPUS=$1" >> /etc/sysconfig/irqbalance
+    fi
+    systemctl try-restart irqbalance
+}
